record-boot-success/record-boot-success

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
+import dbus
+import dbus.service
+import logging
+import os
+import subprocess
+
+from systemd import journal
+
+BUSNAME = "com.endlessm.RecordBootSuccess"
+
+
+def record_boot_success():
+    envfile = "/boot/grub/grubenv"
+    program = "/usr/bin/grub-editenv"
+
+    try:
+        if os.path.getsize(envfile) == 0:
+            os.remove(envfile)
+    except OSError:
+        pass
+
+    cmd = [program, envfile, "unset", "recordfail"]
```

Comment:
What about new PAYG systems that boot with sd-boot? I believe these will use the regular x86_64 ostree, so `grub-editenv` will be executable. But I don't think `/boot/grub` will exist. I bet this will fail.